api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java

```diff
@@ -78,12 +71,8 @@ public double getWorkspaceFreeTierBillingUsage(DbWorkspace dbWorkspace) {
 
   /**
    * Check whether users have incurred sufficient cost in their workspaces to trigger alerts due to
-   * passing thresholds or exceeding limits. RW-6280 - REQUIRES_NEW transactional mode was added to
-   * make the call to this method create a new transaction with each set of users. In order to
-   * commit the transaction after the call. However, if the user has many workspaces, this method
-   * may still timeout.
+   * passing thresholds or exceeding limits.
    */
-  @Transactional(propagation = Propagation.REQUIRES_NEW)
```

Comment:
Transactional is not needed now, it was originally added as per RW-6280 to update the workspace in batches and commit each batch separately. Now that we use a cloud task, the new transaction is not needed